src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+function Calculator() {
+  return (
```

Comment:
- [ ]  Please, consider that the Calculator.js should be a class-based component, as it is required in the project.